server/events/apply_requirement_handler.go

```diff
@@ -0,0 +1,50 @@
+package events
+
+import (
+	"github.com/pkg/errors"
+	"github.com/runatlantis/atlantis/server/core/runtime"
+	"github.com/runatlantis/atlantis/server/events/models"
+	"github.com/runatlantis/atlantis/server/events/yaml/raw"
+	"github.com/runatlantis/atlantis/server/events/yaml/valid"
+)
+
+//go:generate pegomock generate -m --package mocks -o mocks/mock_apply_handler.go ApplyRequirementHandler
+type ApplyRequirementHandler interface {
+	HandleApplyRequirements(repoDir string, ctx models.ProjectCommandContext) (string, string, bool, error)
+}
+
+type DefaultApplyRequirementHandler struct {
```

Comment:
Not a huge fan of naming something Default etiher, it's not super descriptive. If you follow my suggestion above, this could be something like `AggregateApplyRequirements`

Reply:
Done!